test/ext/declarative/test_basic.py

```diff
@@ -28,6 +28,7 @@
 from sqlalchemy.orm import mapper
 from sqlalchemy.orm import properties
 from sqlalchemy.orm import relationship
+from sqlalchemy.orm import close_all as close_all_sessions
```

Comment:
**mike bayer** (zzzeek) wrote:

i think this needs to go up above.   the new pep8 job will note that.  the sorting scheme I'm using is https://github.com/PyCQA/flake8-import-order/blob/master/tests/test_cases/complete_google.py